### PR TITLE
Migrated Travis CI to Github Actions

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Migrated Travis CI to Github Actions
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | N/A
| How to test?      | CI is :green_circle: https://github.com/Progi1984/docker-ci/actions
| Possible impacts? | N/A

### CODE REVIEW

File: .github/workflows/docker.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      run: docker build -t "$image" .
+      working-directory: ${{ matrix.php-versions }}
+
+    - name: Docker Images
+      run: docker images
+      working-directory: ${{ matrix.php-versions }}
+
+    - name: Docker Test
+      run: ~/official-images/test/run.sh "$image"
+      working-directory: ${{ matrix.php-versions }}
```
Comment (PierreRambaud) [inline .github/workflows/docker.yml]:
```suggestion
      working-directory: ${{ matrix.php-versions }}

```

Comment (Progi1984) [reply .github/workflows/docker.yml]: Fixed

File: .github/workflows/docker.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    - uses: actions/checkout@v2
+
+    - name: Fetch Docker Official
+      run: git clone https://github.com/docker-library/official-images.git ~/official-images
+
+    - name: Set environment variable
+      run: echo "image=prestashop/prestashop-git:${{ matrix.php-versions }}" >> $GITHUB_ENV
+
+    - name: Docker Build
+      run: docker build -t "$image" .
```
Comment (PierreRambaud) [inline .github/workflows/docker.yml]:
```suggestion
      run: docker build -t "${{ env.image }}" .
```

File: .github/workflows/docker.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+      working-directory: ${{ matrix.php-versions }}
+
+    - name: Docker Images
+      run: docker images
+      working-directory: ${{ matrix.php-versions }}
+
+    - name: Docker Test
+      run: ~/official-images/test/run.sh "$image"
```
Comment (PierreRambaud) [inline .github/workflows/docker.yml]:
```suggestion
      run: ~/official-images/test/run.sh "${{ env.image }}"
```

**Review by PierreRambaud — COMMENTED** [review]
I'm afraid the variable attribution is wrong regarding the documentation: 
https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable

File: .github/workflows/docker.yml (hunk 3 above)
Comment (Progi1984) [reply .github/workflows/docker.yml]: Fixed

File: .github/workflows/docker.yml (hunk 2 above)
Comment (Progi1984) [reply .github/workflows/docker.yml]: Fixed

**Review by PierreRambaud — APPROVED** [review]

**Review by jolelievre — APPROVED** [review]

**Pull request merged**